linux_os/guide/system/permissions/restrictions/enable_execshield_settings/sysctl_kernel_exec_shield/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,rhel7,rhel8,rhel9,rhv4
+prodtype: fedora,rhel7,rhel8,rhel9,rhv4,sle15
```

Comment:
Please replace `Red Hat Enterprise Linux 7` with `{{{ full_name }}}` so that way the text in the guide and report is accurate for other OSes.

Reply:
Thanks for the input, should be OK in [2b5bc9d](https://github.com/ComplianceAsCode/content/pull/7891/commits/2b5bc9df6e39c95ef9ae81b4281b720cc783d067)